plugin-hrm-form/src/components/CSAMReport/CSAMReportTypePicker.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import { Template } from '@twilio/flex-ui';
+import { useForm } from 'react-hook-form';
+
+import ActionHeader from '../case/ActionHeader';
+import { BottomButtonBar, Box, StyledNextStepButton } from '../../styles/HrmStyles';
+import { BoldDescriptionText, CSAMReportContainer, CSAMReportLayout } from '../../styles/CSAMReport';
+import { getInputType } from '../common/forms/formGenerators';
+import { CSAMReportType } from '../../states/csam-report/types';
+import { externalReportDefinition } from './CSAMReportFormDefinition';
+
+type Props = {
+  renderContactDetails?: boolean;
+  counselor: string;
+  onClickClose: () => void;
+  onSubmit: () => void;
+  isEmpty?: boolean;
+  methods: ReturnType<typeof useForm>;
+  reportType: CSAMReportType;
+  pickReportType: (reportTypeFormValue: string) => void;
+};
+
+const CSAMReportTypePicker: React.FC<Props> = ({
+  counselor,
+  onClickClose,
+  onSubmit,
+  methods,
+  reportType,
+  pickReportType,
+}) => {
+  const formElement = React.useMemo(() => {
+    return getInputType([], () => pickReportType(methods.getValues(['reportType']).reportType))(
+      externalReportDefinition[0],
+    )(reportType);
+  }, [methods, pickReportType, reportType]);
```

Comment:
I believe `methods` is an object reference that changes on every rerender, making useless the `useMemo` here. For simplicity we can get rid of the memo.

```suggestion
  const { getValues } = methods;
  const formElement = React.useMemo(() => {
    return getInputType([], () => pickReportType(getValues(['reportType']).reportType))(externalReportDefinition[0])(
      reportType,
    );
  }, [getValues, pickReportType, reportType]);
```